keys.go

```diff
@@ -342,7 +351,7 @@ type Policies struct {
 }
 
 // GetPolicy retrieves a policy by Key ID.
-func (c *Client) GetPolicy(ctx context.Context, id string) (*Policy, error) {
+func (c *Client) GetPolicy(ctx context.Context, id string) ([]Policy, error) {
```

Comment:
I think maybe you want to add a GetPolicies that returns all key policies and still have a GetPolicy that returns the specific policy type still.

Reply:
yes we do have that as our work item. But currently we are supporting return all policies upon get policies. Plan to add individual policy later.